services/ui-src/src/components/cards/ReportIntroCardActions.tsx

```diff
@@ -13,6 +13,21 @@ export const ReportIntroCardActions = ({ reportType }: Props) => {
   const state = useStore().user?.state;
   const dashboardRoute = `/report/${reportType}/${state}`;
 
+  const getAbbreviation = (reportType: ReportType) => {
+    if (!isReportType(reportType)) return "";
+    switch (reportType) {
+      case ReportType.CI:
+        return "CI";
+      case ReportType.QMS:
+        return "QMS";
+      case ReportType.TA:
+        return "TACM";
+      default:
+        assertExhaustive(reportType);
+        return "";
+    }
+  };
```

Comment:
I wasn't sure about switching the actual ReportType to `TACM`, as that would require (I think) extensive changes throughout the front and backend.
I opted for this switch statement as I have seen it used in a couple of other places, but let me know if someone knows of a more elegant non-intrusive way.